.github/workflows/build.yml

```diff
@@ -4,29 +4,35 @@ on:
   push:
     branches:
       - master
-    paths-ignore:
-      - '.github/workflows/codeql-analysis.yml'
-      - '.github/workflows/fill-gh-draft-release.yml'
-      - '.github/workflows/generate-documentation.yml'
-      - '.github/workflows/release.yml'
-      - '.github/workflows/upload-demo-archive.yml'
-      - '.gitignore'
-      - 'docs/**'
-      - '*.md'
-      - LICENSE
+    paths:
```

Comment:
I was thinking maybe we can use:
on: [push, pull_request]

it will permit to avoid paths duplication

Reply:
@aibcmars @csouchet this is not possible according to https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration

> If you need to specify activity types or configuration for an event, you must configure each event separately. You must append a colon (:) to all events, including events without configuration.

Reply:
ah, pitty